core/arch/arm/plat-vexpress/main.c

```diff
@@ -256,3 +257,14 @@ int psci_cpu_on(uint32_t core_id, uint32_t entry, uint32_t context_id)
 	return PSCI_RET_SUCCESS;
 }
 #endif /*PLATFORM_FLAVOR_qemu_virt*/
+
+#if defined(IT_SEC_PHY_TIMER) && !defined(CFG_CORE_SEL2_SPMC)
+static TEE_Result init_callout_service(void)
+{
+	timer_init_callout_service(interrupt_get_main_chip(), IT_SEC_PHY_TIMER);
```

Comment:
Still wondering about `is_per_cpu` here :wink: what happens when `IT_SEC_PHY_TIMER` is triggered? Is it delivered to only one core (and in this case, always the same or not?), to all cores, to a subset of all cores...?

Reply:
There is one timer per CPU, and each of them only triggers a PPI on that CPU. The PPI is triggered only on the CPU where the generic timer expires.